common/persistence/visibility/store/cassandra/query_filters.go

```diff
@@ -0,0 +1,142 @@
+package cassandra
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/xwb1989/sqlparser"
+	"go.temporal.io/api/enums/v1"
+)
+
+type QueryFilters struct {
+	HasWorkflowIDFilter bool
+	WorkflowID string
+	HasWorkflowTypeFilter bool
+	WorkflowType string
+	HasExecutionStatusFilter bool
```

Comment:
Alternative is to use *string for each column, then we don't need this extra bool flag. But it is your call.

Reply:
Changed to use the existing sqlplugin.VisibilitySelectFilter struct instead.